client/src/plugins/element-templates-modal/__tests__/ElementTemplatesModalSpec.js

```diff
@@ -0,0 +1,230 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+import ElementTemplatesModal from '../ElementTemplatesModal';
+
+chai.should();
+chai.use(sinonChai);

```

Comment:
Do we need that even though we use https://github.com/camunda/camunda-modeler/blob/develop/client/karma.config.js#L67?

Reply:
We don't. I'll remove the unnecessary code.

Reply:
Done.